src/main/scala/exploration/strategies/standardStrategies.scala

```diff
@@ -0,0 +1,192 @@
+package exploration.strategies
```

Comment:
Is this now redundant with (some of the) strategies that have been added to the shine repo as part of the big merge?

Reply:
Yes, but they are defined in the tvmGEMM test class